WooCommerce/Classes/ViewRelated/Dashboard/MyStore/Cells/ProductTableViewCell.xib

```diff
@@ -26,8 +26,8 @@
                                     <constraint firstAttribute="width" secondItem="yk6-X3-NSJ" secondAttribute="height" multiplier="1:1" id="52Z-Kf-0Bw"/>
                                 </constraints>
                             </imageView>
-                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="wbD-Qx-IG5">
-                                <rect key="frame" x="45" y="0.0" width="145" height="58.5"/>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" spacing="3" translatesAutoresizingMaskIntoConstraints="NO" id="wbD-Qx-IG5">
```

Comment:
We try to use [8pt in layout](https://woomobiledesignhandbook.wordpress.com/layout/)  and I wonder if it makes sense to try and stick to the half of that (4pt) instead of 3pt. 

Will it mess up the layout?

Reply:
4pt, looks good too.
However I don't have access to https://woomobiledesignhandbook.wordpress.com/layout/ 😇